tests/x11/gnote/gnote_search_body.pm

```diff
@@ -15,11 +15,13 @@
 use base "x11test";
 use strict;
 use testapi;
+use version_utils 'is_opensuse';
 
 
 sub run {
     my ($self) = @_;
     $self->gnote_launch();
+    send_key "down" if is_opensuse;
```

Comment:
I don't think that this is really an "openSUSE" difference but rather depends on the version of gnote which differs per distribution version